pytensor/link/pytorch/linker.py

```diff
@@ -13,7 +13,11 @@ def input_filter(self, inp: Any) -> Any:
         return pytorch_typify(inp)
 
     def output_filter(self, var: Variable, out: Any) -> Any:
-        return out.cpu()
+        import torch
+
+        if torch.is_tensor(out):
```

Comment:
Put these changes in a separate commit (and squash the Elemwise ones)

Reply:
okay sounds good

Reply:
I got rid of this commit.